packages/twenty-server/src/modules/messaging/monitoring/services/messaging-telemetry.service.ts

```diff
@@ -30,6 +30,7 @@ export class MessagingTelemetryService {
     await this.analyticsService.create(
       {
         type: 'track',
+        sessionId: '',
```

Comment:
We want to keep sending those events to the previous service for now (t.twenty.com) ; this is not analytics but this is more a monitoring service. It would pollute our analytics